frontend/packages/console-shared/src/promql/OWNERS

```diff
@@ -0,0 +1,16 @@
+reviewers:
+  - kyoto
+  - lilic
+  - paulfantom
+  - s-urbaniak
+  - simonpasquier
+  - spadgett
+approvers:
+  - bparees
+  - christianvogt
+  - jhadvig
+  - kyoto
+  - rawagner
+  - rhamilto
+  - spadgett
+  - vojtechszocs
```

Comment:
Most of the approvers are listed in `frontend/OWNERS`, so they don't all need to be listed explicitly again here. I'm happy adding some folks from the monitoring team to approvers for this folder if you like.

Reply:
👍 Updated